src/builder/Actions.sol

```diff
@@ -71,22 +85,30 @@ library Actions {
         uint256 paymentMaxCost;
     }
 
-    struct TransferActionContext {
+    struct BridgeActionContext {
         uint256 amount;
         uint256 price;
         address token;
         uint256 chainId;
         address recipient;
+        uint256 destinationChainId;
+        string bridgeType;
     }
 
-    struct BridgeActionContext {
+    struct CometSupplyActionContext {
+        uint256 amount;
+        uint256 chainId; // ?
```

Comment:
I believe these fields are not present on our initial idea of the supply action context, not sure if that's a problem.